docs/rfc-1-staking-contract.adoc

```diff
@@ -351,25 +351,25 @@ protect against DoSing slashing queue. Can only be called by the governance.
 
 === Stake top-up
 
-==== `topUp(address operator, uint256 amount) external`
+==== `topUp(address operator, uint96 amount) external`
 
 Increases the amount of the stake for the given operator. The sender of this
 transaction needs to have the amount approved to transfer to the staking
-contract. Can be called by anyone.
+contract. Can be called only by the owner or operator.
```

Comment:
To prevent this scenario:
1. Someone wants to unstake keep and calls "unstakeKeep"
2. The user sends the tx for the Keep legacy contract to "undelegate"
3. This tx lands in the men pool and someone front runs it by calling "topUpKeep"
4. The undelegate is mined after the top up
5. The attacker calls the notify keep discrepancy function to slash

Reply:
Makes sense. I missed the fact it's possible to top-up after unstaking, this is not allowed in KEEP staking contract and I have not switched my mindset yet. 

We need to update the RFC then at some point - in this or another PR - I'll leave this discussion unresolved as a reminder for us until we update the RFC.

Reply:
I found in RFC only one line about top-up roles (that you mentioned above) and changed it. Or you want to add more about this?

Reply:
There is also: 

> Anyone can execute a stake top-up for an operator using a liquid T 

in section 2.1.3. Stake top-up.

Reply:
Ok, if only this then changed already